frontend-react/src/hooks/network/History/DeliveryHooks.ts

```diff
@@ -29,12 +36,19 @@ const useOrgDeliveries = (org?: string, service?: string) => {
                 segments: {
                     orgAndService,
                 },
+                params: {
+                    sortDir: filters?.order,
+                    since: filters?.from,
+                    until: filters?.to,
+                    pageSize: filters?.size,
+                    cursor: "",
+                },
             }),
-        [authorizedFetch, orgAndService]
+        [authorizedFetch, orgAndService, filters]
     );
     const { data } = rsUseQuery(
         // sets key with orgAndService so multiple queries can be cached when swapping services
-        [getOrgDeliveries.queryKey, orgAndService],
+        [getOrgDeliveries.queryKey, { orgAndService, filters }],
```

Comment:
So, this bit here basically tells ReactQuery, "Hey, whenever my orgAndService or filters update, refetch, because that data is not cached". This is something of a cache key interpolation so it won't refetch as we apply filters it's previously fetched with; it'll pull those from the cache.